classes/submission/form/SubmissionSubmitStep1Form.inc.php

```diff
@@ -105,4 +105,28 @@ function setSubmissionData($submission) {
 		$submission->setSeriesPosition($this->getData('seriesPosition'));
 		parent::setSubmissionData($submission);
 	}
+
+	/**
+	 * Save changes to submission.
+	 * @return int the submission ID
+	 */
+	function execute(...$functionParams) {
+		
+		$submissionId = parent::execute(...$functionParams);
+
+		$request = Application::get()->getRequest();
+
+		$seriesDao = \DAORegistry::getDAO('SeriesDAO'); /** @var SeriesDAO $seriesDao */
```

Comment:
For the `stable-3_3_0` branch, almost everything is in the root namespace; this `\` is not required.

Reply:
fixed.